app/frontend/src/components/forms/ShareForm.vue

```diff
@@ -107,5 +114,22 @@ export default {
 };
 </script>
 
-<style>
+<style scoped lang="scss">
+@import '~vuetify/src/styles/settings/_variables';
+
+.qrCodeContainer {
+  @media #{map-get($display-breakpoints, 'sm-and-up')} {
+    padding-left: 75px;
+  }
+
+  ::v-deep canvas {
+    margin-top: 50px;
+    max-width: 250px;
+    max-height: 250px;
+  }
+}
+
+.close-dlg {
+  margin-top: 50px;
+}
```

Comment:
Consider using rem units in lieu of px units.

Reply:
All our mockup sizes have spacing in pixels so it's easier to match to those keeping it in pixels (vuetify vars are in pixels for spacing as well). So for text usage I see rem making sense but size and stuff I'd rather keep in pixel.